PowerShell.sublime-syntax

```diff
@@ -292,9 +299,9 @@ contexts:
                 (?xi:
                   \b(
                     Mandatory | ValueFromPipeline(?:ByPropertyName)?
-                  | ValueFromRemainingArguments | Position
+                  | ValueFromRemainingArguments | Position(?:alBinding)?
                   | (?:Default)?ParameterSetName | SupportsShouldProcess | SupportsPaging
-                  | PositionalBinding | HelpUri | ConfirmImpact | HelpMessage
+                  | HelpUri | ConfirmImpact | HelpMessage
                   )
```

Comment:
As following whitespace and `=` is optional, maybe add a word boundary check to ensure not to highlight beginning of unknown args.
```suggestion
                  )\b
```

Reply:
otherwise LGTM!